services/librariesio/librariesio-dependent-repos.service.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const { metric } = require('../../lib/text-formatters')
+const LibrariesIoBase = require('./librariesio-base')
+
+// https://libraries.io/api#project-dependent-repositories
+class LibrariesIoDependentRepos extends LibrariesIoBase {
+  static get category() {
+    return 'downloads'
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'dependent repos',
+    }
+  }
+
+  static get url() {
+    return this.buildUrl('librariesio/dependent-repos')
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Dependent repos (via libraries.io)',
+        previewUrl: 'npm/got',
+      },
+    ]
+  }
+
+  static render({ dependentReposCount }) {
+    return {
+      message: metric(dependentReposCount),
+      color: dependentReposCount === 0 ? 'orange' : 'brightgreen',
```

Comment:
Is this definitely a good/bad (green/orange), or would it be better to think of it as an info (blue)?

Reply:
I based this on the download badges, which are green. I think it's more like that those than like a license badge.